### PR TITLE
update Readme

### PR DESCRIPTION
**Pull request by MichaelClifford**
## Related Issues and Dependencies
None
## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Updated the Readme to give users a better sense of what this project is and how they could either contribute to it or modify it for their own purposes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,66 @@
 # Mailing List Analysis Toolkit
 
-This repository contains an example framework for how to use the Open Data Hub on Openshift to develop a custom end-to-end email analytics service.
+This repository contains example code for how to develop a custom end-to-end email analytics service using the Open Data Hub on OpenShift. Although the specific example here is a mailing list analysis tool, our hope is to show that this approach could be easily modified and adapted to many intelligent application use cases.
+
+#
+## Project Assumptions
+This repository assumes that you have an existing Open Data Hub deployed on OpenShift that includes, JupyterHub, Argo CD, Ceph, Hive, Hue and Superset.
```
Comment (tumido) [inline README.md]:
```suggestion
This repository assumes that you have an existing Open Data Hub deployed on OpenShift that includes, JupyterHub, Argo, Ceph, Hive, Cloudera Hue and Apache Superset.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+## Project Overview
+The purpose of this project is to develop and maintain an open source NLP application that provides regular and up-to-date analytics for large open source development project  mailing lists.
+
+### **Current Lists/ Datasets**
+* [Fedora Devel](https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/)
+* ? (please open an issue if you'd like another mailing list included)
+
+### **Application Overview**
+
+At a high level, this application can be seen as an Argo workflow which orchestrates a set of jupyter notebooks to push transformed data to Ceph. Each notebook is responsible for a single task and is used either for collecting raw data from the Fedora hyperkitty archive (our live data set), preprocessing that data, or performing some specific analytics task. In almost all cases, the notebooks both push their outputs to Ceph remote storage (for use in a future run) as well as maintain a local copy within the application pod for use by other notebook processes. Finally we use external tables in Apache Hive, with Hue, to connect the Ceph data to a SQL database for interactive visualization with Superset.
```
Comment (tumido) [inline README.md]:
```suggestion
At a high level, this application can be seen as an [Argo Workflow](https://argoproj.github.io/argo/) which orchestrates a set of Jupyter notebooks to push transformed data to Ceph. Each notebook is responsible for a single task and is used either for collecting raw data from the [Fedora HyperKitty mailing list archive](https://lists.fedoraproject.org/archives/) (our live data set), preprocessing that data, or performing some specific analytics task. In almost all cases, the notebooks both push their outputs to Ceph remote storage (for use in a future run) as well as maintain a local copy within a shared volume among the application's pods for use by other notebook processes. Finally we use external tables in Apache Hive, with Hue, to connect the Ceph data to a SQL database for interactive visualization with Superset.
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,66 @@
...
+
+![](docs/assets/images/app-overview.png)
+
+
+### **User Interaction**
+
+The primary output and user interface for this application is a [Superset](https://superset.apache.org/) dashboard. This tool allows us to define certain data visualization elements from our analysis that we would like to publish and share with others, while also including enough flexibility and interactivity to allow users to explore the data themselves.
+
+If our application is working correctly, it should regularly and automatically re-run the analyses and ensure that the dashboard and all its plots are current and up to date.
```
Comment (tumido) [inline README.md]:
```suggestion
Our application is designed to automatically re-run the analyses on regular basis and ensure that the dashboard and all its plots are current and up to date.
```

**Review by tumido — CHANGES_REQUESTED** [review]
LGTM, few changes suggested in the comments.

It also reminds me, we need to move the notebooks from `stage` to their appropriate folders next. :slightly_smiling_face:

**Review by tumido — APPROVED** [review]

**Comment by MichaelClifford** [comment]
@tumido thanks! good comments. I committed the suggested changes directly. and yes, I'll be sure we move the notebooks out of stage in an upcoming PR : )

**Pull request merged**